.github/workflows/main.yaml

```diff
@@ -10,9 +13,16 @@ jobs:
       run: |
         sudo apt-get update
         sudo apt-get install -y --no-install-recommends doxygen doxygen-latex
+    - name: Get UCX
+      run: git clone ${OPEN_UCX_LINK} -b ${OPEN_UCX_BRANCH} /tmp/ucx
+    - name: Build UCX
+      run: |
+        cd /tmp/ucx && ./autogen.sh
+        ./contrib/configure-release --without-java --disable-numa --prefix $PWD/install
+        make -j install
     - uses: actions/checkout@v1
     - name: Build UCC
       run: |
         ./autogen.sh
```

Comment:
@alex--m i'm not sure i understand what you mean? You want to have "with-ucg" as autogen option? Why is it simpler? Do you want to add it  right now in the CI?

Reply:
For the time being, UCG is an optional part of UCX (master). To build UCX with UCG - one needs to pass `--with-ucg` to autogen and `--enable-ucg` to configure. Long-term this will change, but short-term I don't see any other way to integrate UCG code into UCC. Specifically, UCG uses a lot of internal UCX APIs (like stats.h and profiling.h) which UCX does not expose publically (and might never will). If you don't mind, I recommend adding both to the UCC CI, otherwise the CI will not be able to test the "reactive" component any time soon...

Reply:
@alex--m lets add ucg related flags when we have the corresponding component, as separate PR - right now there is no ucg detection in configure.

Reply:
I suppose that's ok.